packages/itwinui-css/src/tabs/borderless.scss

```diff
@@ -34,12 +31,6 @@
     border-inline-start: var(--iui-size-3xs) solid var(--iui-color-border);
```

Comment:
Can we decrease the base thickness of the line so that the active stripe stands out more? `1px` should be good.

```css
  .iui-horizontal & ~ .iui-tabs-content {
    margin-block-start: -1px;
    border-block-start: 1px solid var(--iui-color-border);
  }

  .iui-vertical & ~ .iui-tabs-content {
    margin-inline-start: -1px;
    padding-inline-start: var(--iui-size-s);
    padding-inline-end: var(--iui-size-s);
    border-inline-start: 1px solid var(--iui-color-border);
  }
```

Reply:
Done in https://github.com/iTwin/iTwinUI/pull/2541/commits/39f8f0b82d3c5cfea89a09eca70cf5b6589ba42a